### ISSUE TITLE
Brianstorm on how we can reduce the amount of sprite re-do for each weapon, to possibly create stuff more swiftly (but still efficiently) and not burden working sprite artist over it

### ISSUE THREAD

**Issue by Pseudo-64** [Team-Awkward-Smile/Epilogue#205]

**Comment by vitormaduro** [comment]
There are a few different ways to reuse sprites and animations:
- Split Hestmor's sprite in 2: one for her lower body, and another for her upper body. This would allow us to move her legs and arms independently of one another, without having to create one animation for each State in which her arms and legs can move (basically how it's done in 3D, by blending the legs animation with the arms animation to create a custom new animation by code)
- Separate the guns in categories, each one with it's own animation for holding them. Doing this would allow us to change Hestmor's sprite to the correct stance and attach the gun to her hand without needing a whole sprite set for each gun. For example, if a pistol and a machine gun are both held with one hand and have the same shooting animation, we can create the sprite sheet for "holding light gun with one hand" and use it for any gun that fits this description. The guns themselves would be a separate object attached to Hestmor by code
- If we take the idea of guns that look like decorations on Hestmor's body, we can take it even further and just attach the gun to Hestmor and keep her with the same animations. Maybe a new attack animation will be needed when shooting, but if a gun is, for example, tied around her arm, most of her animation will remain the same, but with a new object attached to her arm
- There's also that software (Spine) suggested by Skaers. It's not cheap and would require some time for everyone (artists and programmers) to get used to the new workflow, but the features it offers are so useful that the time we spend learning it would be rewarded tenfold by not needing to create a bunch of animations for each state
- Apart from those things, I can't think of anything else that may help us, but I'll update this issue if anything else comes up

TL;DR:
- Split Hestmor in two
- Guns are separate objects attached by code. Most of them should be held the same way
